eng/automation/changelog/src/main/java/com/azure/resourcemanager/tools/changelog/changelog/ChangeLog.java

```diff
@@ -24,20 +26,19 @@
 public class ChangeLog {
     private AllMethods allMethods;
     protected List<String> newFeature;
-    protected List<String> breakingChange;
-
-    ChangeLog() {
-        this.newFeature = new ArrayList<>();
-        this.breakingChange = new ArrayList<>();
-    }
+    protected BreakingChange breakingChange;
 
     ChangeLog(AllMethods allMethods) {
         this.allMethods = allMethods;
         this.newFeature = new ArrayList<>();
-        this.breakingChange = new ArrayList<>();
+        this.breakingChange = BreakingChange.fromClass(getJApiClass().getFullyQualifiedName());
```

Comment:
nit, would `BreakingChange.onJavaClass` be better for understanding? Or maybe just a plain ctor?
(I was confusing this with initialize class via reflection)

Reply:
Yeah, I'll change the method name.

Reply:
changed